app/src/test/java/com/codesoom/assignment/models/TaskTest.java

```diff
@@ -0,0 +1,59 @@
+package com.codesoom.assignment.models;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class TaskTest {
+
+    @DisplayName("UpdateTitle Method")
+    @Nested
+    class Describe_updateTitle {
+        @DisplayName("if an updated task is given")
+        @Nested
+        class Context_with_update_title {
+            private Task task;
+
+            @BeforeEach
+            void setup() {
+                task = new Task(1L, "title");
+            }
```

Comment:
```suggestion
           private final Task task = new Task(1L, "title");
```

Task가 불변이고 계속 상태라 바꿔서 사용하지 않을 것이라면 그대로 써도 되지 않을까해요!